packages/atclient/src/atkey.c

```diff
@@ -212,79 +214,85 @@ exit: {
 }
 }
 
-int atclient_atkey_to_string(const atclient_atkey atkey, char *atkeystr, const unsigned long atkeystrlen,
+size_t atclient_atkey_strlen(const atclient_atkey *atkey) {
+  size_t len = 0;
+
+  return len;
+}
+
+int atclient_atkey_to_string(const atclient_atkey *atkey, char *atkeystr, const unsigned long atkeystrlen,
```

Comment:
atkey as a pointer, strlen function is a WIP

Reply:
Sorry, I thought this was also done, will do this with my monitor implementation.